File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        
        
        if len(s) != len(t):
            return False
        char = [0 for _ in range(0,26)]
        n = len(s)
        for i in range(n):
            char[ord(s[i]) - ord('a')] +=1
            char[ord(t[i]) - ord('a')] -=1
        for j in char:
            if j != 0:
                return False
        return True

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {

        int[] chars = new int[26];
        int n = s.length();
        int m = t.length();
        if(n!=m){
            return false;
        }
        for(int i = 0; i < n; i++){
            int a = s.charAt(i);
            int b = t.charAt(i);
            chars[a-97] +=1;
            chars[b-97] -=1;

        }
        for(int nums : chars){
        if(nums != 0){
            return false;
        }
        }
        return true;
    }
    
}